controls/V-81015.rb

```diff
@@ -0,0 +1,47 @@
+# encoding: utf-8
+#
+control "V-81015" do
+  title "The Red Hat Enterprise Linux operating system must be configured to use the au-remote plugin."
+  desc  "
+  Information stored in one location is vulnerable to accidental or incidental deletion or alteration. Off-loading is 
+  a common process in information systems with limited audit storage capacity. Without the configuration of 
+  the \"au-remote\" plugin, the audisp-remote daemon will not off-load the logs from the system being audited.
+  "
+  impact 0.5
+  tag "gtitle": "SRG-OS-000342-GPOS-00133"
+  tag "satisfies": ["SRG-OS-000342-GPOS-00133", "SRG-OS-000479-GPOS-00224"]
+  tag "gid": "V-81015"
+  tag "rid": "SV-95727r1_rule"
+  tag "stig_id": "RHEL-07-030200"
+  tag "cci": ["CCI-001851"]
+  tag "documentable": false
+  tag "nist": ["AU-12 c", "Rev_4"]
+  tag "subsystems": ["audit"]
+  tag "check_id": "C-80729r1_chk"
+  tag "fix_id": "F-87849r2_fix"
+  desc "check", "
+  Information stored in one location is vulnerable to accidental or incidental deletion or alteration.
+
+  Off-loading is a common process in information systems with limited audit storage capacity.
+
+  Without the configuration of the \"au-remote\" plugin, the audisp-remote daemon will not off-load the logs from the system being audited.
+  "
+  desc "fix", "
+  Edit the /etc/audisp/plugins.d/au-remote.conf file and change the value of \"active\" to \"yes\".
+
+  The audit daemon must be restarted for changes to take effect:
+
+  # service auditd restart
+  "
+
+  if file('/etc/audisp/plugins.d/au-remote.conf').exist?
+    describe parse_config_file('/etc/audisp/plugins.d/au-remote.conf') do
+      its('active') { should match %r{yes$} }
+    end
+  else
+    describe "File '/etc/audisp/plugins.d/au-remote.conf' cannot be found. This test cannot be checked in a automated fashion and you must check it manually" do
+      skip "File '/etc/audisp/plugins.d/au-remote.conf' cannot be found. This check must be performed manually"
+    end
```

Comment:
It would be interesting to see what additional behavior is acceptable, in this case.   The STIG does not allow for additional behavior, so I am uncertain why you are coding manual review as optional.  This appears to be a clear pass/fail check.   I would remove the if/else control structure and just search for the file.

Now the question is, if what you are intending is valid.  That is, are there secondary approved options. If this is the case, I would just say implement them here.   Give an attribute in that control structure to require organizational documentation stating why this option is valid for their use case.  In the end, I would say this is unneeded.  What I would expect is exceptions, the likes of which you are allowing to be manually reviewed, to be overridden by organizational inheritance.

Reply:
Agree on this one. The STIG should be modeled exactly. There is no contingency for the file not being there. If it does not exist or does not contain the prescribed value this is a fail.